### PR TITLE
fix: minor tailwind issues

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Found a couple more tailwind issues.

I ran through all screens and modals I could test locally, and didn't see anything other weirdly out of place.
However do notify me as soon as you see anything odd *(or unsure off)

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

File: packages/shared/src/components/fields/MarkdownInput/index.tsx [hunk 1]
```diff
@@ -278,7 +280,7 @@ function MarkdownInput(
             wrapper={(children) => (
               <span
                 className={classNames(
-                  'grid grid-cols-3 gap-3',
+                  'flex gap-3',
```
Comment (sshanzel) [inline packages/shared/src/components/fields/MarkdownInput/index.tsx:283]: Just a question, why did we end up moving to `flex gap`, is the `grid grid-cols-n` doesn't work as how it used to anymore?

**Review by sshanzel — APPROVED** [review]
Looks good to me! 🚀

File: packages/shared/src/components/fields/MarkdownInput/index.tsx (hunk 1 above)
Comment (rebelchris) [reply packages/shared/src/components/fields/MarkdownInput/index.tsx:283]: I think this was a fault to be honest.
Don't know why that was chosen, but on one screen we only render 1 item, which made it align super weird (in the first of 3 columns)

Flex does exactly this, but automatic 😅
Love to hear if it has a particular use-case I'm unaware off

Comment (sshanzel) [reply packages/shared/src/components/fields/MarkdownInput/index.tsx:283]: Nothing really. Was just curious about the change if it was tailwind update related. Flex does the job better anyways, and won't limit us to the number of elements. Way to go! 🚀

**Pull request merged**